provider/vsphere/vm_template.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2021 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package vsphere
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"path"
+	"strings"
+
+	"github.com/juju/collections/set"
+	"github.com/juju/errors"
+	"github.com/juju/juju/environs"
+	"github.com/juju/juju/environs/imagemetadata"
+	"github.com/juju/juju/provider/common"
+	"github.com/juju/juju/provider/vsphere/internal/vsphereclient"
+
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/vim25/types"
```

Comment:
"GitHub.com/juju/juju/..." belong in the last import stanza.

Everything else in the selection, belongs in the 2nd import stanza